Platform/RaptorlakeBoardPkg/BoardConfigRpls.py

```diff
@@ -22,9 +22,9 @@ def __init__(self, *args, **kwargs):
 
         super(Board, self).__init__(*args, **kwargs)
 
-        self.VERINFO_IMAGE_ID     = 'SB_RPLS'
+        self.VERINFO_IMAGE_ID     = 'SB_RPLSR'
         self.VERINFO_PROJ_MAJOR_VER = 1
-        self.VERINFO_PROJ_MINOR_VER = 6
+        self.VERINFO_PROJ_MINOR_VER = 5
```

Comment:
It is confusing to change the version to a small number that is used in RPLS MR5.
It is better to increase the number to 7.